ietf/doc/tests_draft.py

```diff
@@ -886,7 +886,11 @@ def test_repair_dead_on_expire(self):
             dead_from_expires.append(d)
             dead_from_expires_event_count[d] = d.docevent_set.count()
 
-        empty_outbox()
+        notified_during_factory_work = mock_notify.call_count
+        for call_args in mock_notify.call_args_list:
+            e = call_args.args[0]
```

Comment:
Wow! I had missed that `call_args` now has `args` and `kwargs` properties! That is so much better than `call_args[0][0]`.